{
  try {
    this.asyncManager.startCallableProcessing(null);
    fail("Expected exception");
  }
 catch (  IllegalArgumentException ex) {
    assertEquals(ex.getMessage(),"Callable is required");
  }
}
